hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -226,4 +230,15 @@ <h2 class="formH2" id="faq">Application FAQs</h2>
         </div>
     </div>
 </div>
+{% endblock %}
+
+{% block scripts %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/qrious/4.0.2/qrious.min.js"></script>
+    <script type="text/javascript">
+        var qrcode = new QRious({
+            element: document.getElementById("qrcode"),
+            value: "{{ user.first_name }};{{ user.last_name }};{{ user.email }}",
```

Comment:
So for newhacks I think data was encoded the same way in the qr code. But also, I don't think this is unsafe because only the participant will have their qr code and first/last name and email is not sensitive data (in my opinion but let me know why you think it is). I don't see how people's information will be exposed in any way. Also encryption on the frontend is not super useful because ppl can find out the encryption algorithm and decrypt it.